### ISSUE TITLE
Release Scheme issues

### ISSUE THREAD

**Issue by medlmichael** [oney/react-native-webrtc#127]
Hi,

Love the library and think it's been a big help to us.  

We've been working with the app and now when device testing we are getting some odd issue when using the release config.  We are using the latest version for react-native on iOS.  We've read through your posts and modified our iOS to disable dead code stripping (project and target of our project).  Your library when looking at it still has then as YES.  Is that something that could be causing the issue?

Is there anything related to SSL issues that might happen in the release config?  Also, we set allow arbitrary loads to true.  The issue only happens in release where the view appears but the initial video is a blank view.

Many thanks!
- Michael

**Comment by lyubomir** [comment]
We build jitsi/jitsi-meet-react in Release on each commit and the issue that you describe doesn't ring a bell for me. Have you tried our fork of react-native-webrtc at https://github.com/jitsi/react-native-webrtc/tree/lib-jitsi-meet?

**Comment by medlmichael** [comment]
no but I'm definitely going to.

The more I've learned it seems that debug seems to always work fine.  When we use release scheme and archive they seem to be less reliable.  ATS has been set to allow for arbitrary loads but still doesn't solve the issue.  To make builds more reliable we had to actually go into RCTWebSocket.m on line 554 and comment out the #IF_DEBUG and end so that this line runs [SSLOptions setValue:@NO forKey:(__bridge id)kCFStreamSSLValidatesCertificateChain];

It's basically overriding something to allow for more forgiving SSL.  Not sure if this is something you have dealt with or if it is more server side so any advice would be great.  Been watching your YouTube vids and they are very helpful.  Thank you.
